pkg/agent/manager/sync.go

```diff
@@ -76,11 +82,14 @@ func (m *manager) processTaintedAuthorities(ctx context.Context, bundle *spiffeb
 		}
 	}
 
-	newTaintedJWTAuthorities := getNewItems(m.processedTaintedJWTAuthorities, jwtAuthorities)
+	newTaintedJWTAuthorities := getNewItemsFromMap(m.processedTaintedJWTAuthorities, jwtAuthorities)
 	if len(newTaintedJWTAuthorities) > 0 {
-		m.c.Log.WithField(telemetry.SubjectKeyIDs, strings.Join(newTaintedJWTAuthorities, ",")).
-			Debug("New tainted JWT authorities found")
-		// TODO: IMPLEMENT!!!
+		// Taint JWT-SVIDs in the cache
+		m.cache.TaintJWTSVIDs(ctx, jwtAuthorities)
```

Comment:
can you keep the log to display that new tainted keys found? that will simplify debugging  tainted authorities propagation